### PR TITLE
Update sample descriptions.

### LINKED ISSUE #350: Add constraint descriptions for the Forseti bundle
The policy bundle docs include a page for all bundles that lists the constraints and a description of each. Add a description for each Forseti constraint missing a description.

### PR DESCRIPTION
**Pull request by gkowalski-google**
Resolves #350

### CODE REVIEW

File: samples/appengine_location.yaml [hunk 1]
```diff
@@ -2,11 +2,13 @@ apiVersion: constraints.gatekeeper.sh/v1alpha1
 kind: GCPAppengineLocationConstraintV1
 metadata:
   name: allow_appengine_applications_in_australia_and_south_america
+  annotations:
+    description: Locations (regions) where App Engine applications are deployed.
```
Comment (morgante) [inline samples/appengine_location.yaml:6]:
```suggestion
    description: Restrict locations (regions) where App Engine applications are deployed.
```

File: samples/appengine_versions.yaml [hunk 2]
```diff
@@ -18,6 +18,8 @@ metadata:
   name: service_versions
   annotations:
     bundles.validator.forsetisecurity.org/scorecard-v1: operational-efficiency
+    description: Tests for amount App Engine application simultaneous versions are
```
Comment (morgante) [inline samples/appengine_versions.yaml:21]:
```suggestion
    description: Limit the number App Engine application versions simultaneously running.
```

File: samples/bigquery_world_readable.yaml [hunk 3]
```diff
@@ -21,7 +21,8 @@ metadata:
     bundles.validator.forsetisecurity.org/cis-v1.1: 5.03
     bundles.validator.forsetisecurity.org/forseti-security: v2.26.0
     bundles.validator.forsetisecurity.org/scorecard-v1: security
+    description: Checks if BigQuery datasets have permissions to assigned to allUsers
```
Comment (morgante) [inline samples/bigquery_world_readable.yaml:24]:
```suggestion
    description: Checks if BigQuery datasets are publicly readable
```

File: samples/gke_restrict_client_auth_methods.yaml [hunk 4]
```diff
@@ -21,6 +21,8 @@ metadata:
     bundles.validator.forsetisecurity.org/scorecard-v1: security
     # This constraint is not certified by CIS.
     bundles.validator.forsetisecurity.org/cis-v1.1: 7.12
+    description: Checks that client certification and password authentication methods
```
Comment (morgante) [inline samples/gke_restrict_client_auth_methods.yaml:24]:
```suggestion
    description: Checks that client certificate and password authentication methods
```

File: samples/restrict_fw_rules_rdp_world_open.yaml [hunk 5]
```diff
@@ -21,16 +21,17 @@ metadata:
   annotations:
     bundles.validator.forsetisecurity.org/scorecard-v1: security
     bundles.validator.forsetisecurity.org/cis-v1.1: 3.07
+    description: Checks that every firewall rule matches certain settings.
```
Comment (morgante) [inline samples/restrict_fw_rules_rdp_world_open.yaml:24]:
```suggestion
    description: Checks for open firewall rules allowing RDP from the internet.
```

File: samples/restrict_fw_rules_world_open.yaml [hunk 6]
```diff
@@ -21,14 +21,15 @@ metadata:
   annotations:
     bundles.validator.forsetisecurity.org/forseti-security: v2.26.0
     bundles.validator.forsetisecurity.org/scorecard-v1: security
+    description: Checks that every firewall rule matches certain settings.
```
Comment (morgante) [inline samples/restrict_fw_rules_world_open.yaml:24]:
```suggestion
    description: Checks for open firewall rules allowing ingress from the internet.
```

File: samples/restrict_fw_rules_ssh_world_open.yaml [hunk 7]
```diff
@@ -21,16 +21,17 @@ metadata:
   annotations:
     bundles.validator.forsetisecurity.org/scorecard-v1: security
     bundles.validator.forsetisecurity.org/cis-v1.1: 3.06
+    description: Checks that every firewall rule matches certain settings.
```
Comment (morgante) [inline samples/restrict_fw_rules_ssh_world_open.yaml:24]:
```suggestion
    description: Checks for open firewall rules allowing SSH from the internet.
```

File: samples/restrict_fw_rules_world_open_tcp_udp_all_ports.yaml [hunk 8]
```diff
@@ -20,23 +20,24 @@ metadata:
   name: restrict-firewall-rule-world-open-tcp-udp-all-ports
   annotations:
     bundles.validator.forsetisecurity.org/forseti-security: v2.26.0
+    description: Checks that every firewall rule matches certain settings.
```
Comment (morgante) [inline samples/restrict_fw_rules_world_open_tcp_udp_all_ports.yaml:23]:
```suggestion
    description: Checks for open firewall rules allowing TCP/UDP from the internet.
```

File: samples/vpc_sc_project_perimeter_v1_blacklist.yaml [hunk 9]
```diff
@@ -16,6 +16,8 @@ apiVersion: constraints.gatekeeper.sh/v1alpha1
 kind: GCPVPCSCProjectPerimeterConstraintV1
```
Comment (morgante) [inline samples/vpc_sc_project_perimeter_v1_blacklist.yaml:16]: We could probably just remove this sample entirely?

Comment (gkowalski-google) [reply samples/vpc_sc_project_perimeter_v1_blacklist.yaml]: Removed.

**Comment by gkowalski-google** [comment]
@morgante Addressed your comments, thanks! PTAL

**Pull request merged**